components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabsTrayStyling.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.tabstray
+
+const val DEFAULT_ITEM_BACKGROUND_COLOR = 0xFFFFFFFF.toInt()
+const val DEFAULT_ITEM_BACKGROUND_SELECTED_COLOR = 0xFFFF45A1FF.toInt()
+const val DEFAULT_ITEM_TEXT_COLOR = 0xFF111111.toInt()
+const val DEFAULT_ITEM_TEXT_SELECTED_COLOR = 0xFFFFFFFF.toInt()
+
+/**
+ * Tabs tray styling for items in the [TabsAdapter]. This can be used as well if you have use a custom [TabViewHolder]
```

Comment:
nit: if you have *to* use? :) or just "if a custom ... is used".

Reply:
"I prefer if you didn't, but if you insist.."

Reply:
:)